src/admin-views/attendees.php

```diff
@@ -90,16 +102,17 @@
 
 					<ul>
 						<?php
+						/** @var Tribe__Tickets__Ticket_Object $ticket */
 						foreach ( $tickets as $ticket ) {
-							/** @var Tribe__Tickets__Ticket_Object $ticket */
+							$ticket_name = sprintf( '%s [#%d]', $ticket->name, $ticket->ID );
 							?>
-						<li>
-							<strong><?php echo esc_html( $ticket->name ) ?>:&nbsp;</strong><?php
-							echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
-						?></li>
-					<?php } ?>
+							<li>
+								<strong><?php echo esc_html( $ticket_name ) ?>:&nbsp;</strong><?php
+								echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
+								?></li>
```

Comment:
:no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 91db2997e16727019f37b4a71b12cafa</sub>